### ISSUE TITLE
Add url for a unit to repository configuration

### ISSUE THREAD

**Issue by mejackreed** [projectblacklight/arclight#591]
Spun out of #541 , add a `url` field for a repository.

This could then be used for display or linking elsewhere.

Design around that TBD.p

**Comment by mejackreed** [comment]
I think this is resolved technically in #612 until there is additional design to add this to the UI. Dropping a design needed on it.

**Comment by ggeisler** [comment]
@mejackreed I'm not sure there is design appropriate for this in the core ArcLight app. If I understand correctly, #612 enables an implementor to add new, arbitrary fields to the `repositories.yml` file. This current ticket asks for a "unit url" field in the repository configuration, which could be one of these new, arbitrary fields added to the `repositories.yml` file.

Since these new fields are not required and we can't anticipate what they are for it is hard to design for them, other than a suggestion I made elsewhere about using the field name of any additional repository fields (un-snake-casing it) followed by the value and just presenting all the additional fields in that form in a list on the Access tab. I believe in that discussion we decided we'd not do that (which is fine with me, I don't think that is a great solution) and it would just be up to the implementor to override whatever views they needed to to display the new fields they chose to add to the `repositories.yml` file.

So I'd suggest we can close this ticket, but let me know if I'm missing something and you do think there is some way we should be able to design a way for these additional repository fields to be displayed in the core ArcLight app.
